test/PandoraInterface.cxx

```diff
@@ -1471,6 +1577,9 @@ bool ParseCommandLine(int argc, char *argv[], Parameters &parameters)
             case 'd':
                 parameters.m_sensitiveDetName = optarg;
                 break;
+            case 'M':
```

Comment:
You also need to explain what the `M` option is in the `PrintOptions()` function. We could also set several of the main parameter combinations with an `-a <integer>` run option (or similar)?